src/hotspot/share/runtime/deoptimization.hpp

```diff
@@ -40,6 +40,32 @@ class compiledVFrame;
 
 template<class E> class GrowableArray;
 
+class DeoptimizationScope {
+ private:
+  // What gen we have done the deopt handshake for.
+  static uint64_t _committed_deopt_gen;
+  // What gen to mark a method with, hence larger than _committed_deopt_gen.
+  static uint64_t _active_deopt_gen;
+  // Indicate a in-progress deopt handshake.
```

Comment:
nit typo: s/a in-progress/an in-progress/

Reply:
Fixed, thanks!